router_configuration_backups/br-mdf-01

```diff
@@ -0,0 +1,385 @@
+## Last changed: 2019-10-27 02:59:16 UTC
+version 15.1X49-D70.3;
+system {
+    host-name br-mdf-01;
+    root-authentication {
+        encrypted-password "$5$.2JOEO0i$GaaUUIlcgelNVJ7qhfu1jnISVcALe0CtWAoRdn.H1a9";
```

Comment:
Can we scrub these from the commit history? Even though we wont be using them Id rather just not include them to at all

Reply:
A little confused. These are the historical configs from the 3 srx routers. I might need to do some cleanup if I have them in more than one place (oops) but I do want to keep records of them in the commit history.

Reply:
Sorry I was talking about the actual line here for `encrypted-password` I know well be rotating this for 18x but still Id rather just have it be junk vs something real

Reply:
Hey. Sure. I can do that later tonight.

Reply:
OK... Added a commit removing the passwords.